src/components/rainbow/__spec__.js

```diff
@@ -63,7 +62,7 @@ describe('Rainbow', () => {
 
     describe('when data changes', () => {
       it("calls setData", () => {
-        const newData = Immutable.fromJS({ foo: "bar" });
+        const newData = fromJS({ foo: "bar" });
```

Comment:
I don't think this had to change, tried it on my machine and it works fine with what it was originally.

Reply:
Yes you are right on this. Just changed it back.